domains/hairon02.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "hairon02",
+    "email": "haironl072@gmail.com"
+  }, 
+  "record": {
+    "CNAME": "hairon02.github.io"
```

Comment:
404 Site. Please send over a preview. Reopen this PR when you have this resolved.